client/app/components/Parameters.less

```diff
@@ -14,13 +14,14 @@
     height: 36px;
   }
 
-  .parameter-container.sortable-container & {
+  .parameter-container.sortable-container &:not(.parameter-dragged) {
```

Comment:
Why change this styling? I did notice there are differences from what's applied to the Dashboard and Query vs what's applied to Widget (not a big difference though), so you may wanna confirm whether the behavior you wanted works for all the cases.

Reply:
This margin aligns the draggable to it's siblings when applied uniformly. Having this addition makes the element one drags "pop", ie offset, in a way that makes it clearer to the user. I found it a nice addition to the user interface.

Reply:
It does cause an interesting effect, but one thing to notice is that it also causes the cursor to be slightly off the drag handle. I think it's better to update it in a follow-up if the idea is to update the styling and not to fix an issue (maybe add some shadow to cause an elevation effect as well).